src/components/ProgramForm/ThreeColProgram/List/index.css

```diff
@@ -163,3 +162,59 @@
   text-align: center;
   transform: translate(-50%, -50%);
 }
+
+/* RECOMMENDER */
+.recommender {
+  position: relative;
+  display: flex;
+  gap: 8px;
+
+  margin: 8px;
+  padding: 14px 14px 14px 0;
+  background-color: #dbfddc;
```

Comment:
I would create custom CSS var for this. (in the recommender block, so in this class)